content/en/contribute.md

```diff
@@ -0,0 +1,102 @@
+can ask on the [mailing
+list](https://mail.python.org/mailman3/lists/scipy-dev.python.org/) or
+[GitHub](http://github.com/scipy/scipy) (open an
+[issue](https://github.com/scipy/scipy/issues) or comment on a relevant
+issue).
+
+Those are our preferred channels (open source is open by nature).
+
+We also have a biweekly _community call_, details of which are announced on the
+[mailing list](https://mail.python.org/mailman3/lists/scipy-dev.python.org/).
+You are very welcome to join.
+If you are new to contributing to open source, we also highly recommend reading
+[this guide](https://opensource.guide/how-to-contribute/).
+
+Our community aspires to treat everyone equally and to value all contributions.
+We have a
+[Code of Conduct](http://scipy.github.io/devdocs/dev/conduct/code_of_conduct.html)
+to foster an open and welcoming environment.
+
+### Writing code
+
+Programmers, this
+[guide](https://scipy.github.io/devdocs/dev/contributor/development_workflow.html#development-workflow)
+explains how to contribute to the codebase.
+
+### Reviewing pull requests
+The project has hundreds of open pull requests -- meaning many potential
+improvements and many open-source contributors waiting for feedback. If you're
+a developer who knows SciPy, you can help even if you're not familiar with the
+codebase. You can:
+* summarize a long-running discussion
+* triage documentation PRs
+* test proposed changes
+
+
+### Developing educational materials
+
+SciPy's [User Guide](https://scipy.github.io/devdocs/tutorial/index.html)
+deserves attention.
+We're in need of new tutorials, how-to's, and deep-dive explanations, and the
+site needs restructuring. Opportunities aren't limited to writers. We'd also
+welcome worked examples, notebooks, and videos.
+
+
+### Issue triaging
+
+The [SciPy issue tracker](https://github.com/scipy/scipy/issues) has a _lot_
+of open issues. Some are no longer valid, some should be prioritized, and some
+would make good issues for new contributors.  You can:
+
+* check if older bugs are still present
+* find duplicate issues and link related ones
+* add good self-contained reproducers to issues
+* label issues correctly (this requires triage rights -- just ask)
+
+Please just dive in.
+
+
+### Website development
+
+We've just revamped our website, but we're far from done. If you love web
+development, these
+[issues](https://github.com/scipy/scipy.org/issues)
+list some of our unmet needs -- and feel free to share your own ideas.
+
+
+### Graphic design
+
+We can barely begin to list the contributions a graphic designer can make here.
+Our docs are parched for illustration; our growing website craves images --
+opportunities abound.
+
+
+### Fundraising
+
+SciPy was all-volunteer for many years, but as its importance grew it became
+clear that to ensure stability and growth we'd need financial support. [This
+SciPy'19 talk](https://www.youtube.com/watch?v=dBTJD_FDVjU) explains how much
```

Comment:
I don't know if this talk is what you wanted here :)

Reply:
Although it's more focused on NumPy, I think it explains well why funding is important. But I could also remove this sentence if you think it's not appropriate (or adjust the timestamp to link to the end of the presentation).

That's something we could have on the scientific-python website itself. cc @jarrodmillman

Reply:
Adding the timestamp would probably be helpful.  We should also continue this conversation over the next year.  We definitely will work on this as part of the Scientific Python project.  I will follow up soonish.